src/Settings.php

```diff
@@ -139,7 +139,7 @@ public function __construct()
                         'attributes' => [
                             'type' => 'text',
                         ],
-                        'desc' => __('Add default button text which appears on <b>Take Action</b> card of <b>Take Action Covers</b> block. <br>Also it would be used for Take Action Cards inside Posts and Take Action Cards in search results', 'planet4-master-theme-backend'),
+                        'desc' => __('Add default button text which appears on <b>Take Action</b> card of <b>Take Action Covers</b> block. <br>Also it would be used for Take Action Cards inside Posts and Take Action Cards in search results', 'planet4-master-theme-backend'), // phpcs:ignore
```

Comment:
It would be nice if we also add which CS rule(`Generic.Files.LineLength.MaxExceeded`) we 
 are ignoring here.